ogarniacz_mp3.py

```diff
@@ -76,6 +76,20 @@ def remove_duplicates(processing_dir_path):
                                 continue # skoro tego pliku nie ma, to nie ma co tutaj robić
 
 
+def adjust_bitrates(processing_dir_path):
+    for dir_path, sub_dirs, files in os.walk(processing_dir_path):
+        for file_name in files:
+            file_path = os.path.join(dir_path, file_name)
+            if classes.MusicFile.is_music_file(file_path):
+                bitrate = classes.MusicFile.__getattr__("bit_rate")
+                if bitrate > 142000:
+                    output_path = os.path.join(dir_path, "tym" + file_name)
+                    proc = subprocess.run(["ffmpeg", "-i", file_path, "-vn", "-ar", "44100", "-ac", "2", "-b:a", "142000",
+                                           output_path], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                    with open(file_path, "w") as f:
```

Comment:
O ile da się przekopiować plik używając open, jest to bardzo nieefektywne. Użyj `shutil.copy` a potem usuń plik tymczasowy.

Na marginesie - gdybyś chciała użyć open, to trzeba by to zrobić tak:
```
with open(file_path, "wb") as f_out:  # wb bo write binary - bo to nie są pliki tekstowe
    with open(output_path, "rb") as f_in:
        content = f_in.read()
    f_out.write(content)
```
A jest to nieefektywne, bo (poza tym że mało czytelne) powoduje wczytanie przez program do pamięci całego kopiowanego pliku. To zwyczajnie dłużej trwa i wymaga więcej RAM-u (pamięci). Tymczasem `shutil.copy` zleci przekopiowanie pliku systemowi, więc twój program będzie potrzebował mniej zasobów.